packages/paste-core/components/side-panel/stories/index.stories.tsx

```diff
@@ -1,28 +1,36 @@
+import { Anchor } from "@twilio-paste/anchor";
+import { Avatar } from "@twilio-paste/avatar";
 /* eslint-disable eslint-comments/disable-enable-pair */
 /* eslint-disable react/jsx-max-depth */
 /* eslint-disable import/no-extraneous-dependencies */
 import { Box } from "@twilio-paste/box";
 import { Button } from "@twilio-paste/button";
+import { ChatComposer } from "@twilio-paste/chat-composer";
+import { ChatBubble, ChatLog, ChatMessage, ChatMessageMeta, ChatMessageMetaItem } from "@twilio-paste/chat-log";
 import { CustomizationProvider } from "@twilio-paste/customization";
+import { DetailText } from "@twilio-paste/detail-text";
 import { Heading } from "@twilio-paste/heading";
+import { ArtificialIntelligenceIcon } from "@twilio-paste/icons/esm/ArtificialIntelligenceIcon";
 import { ChevronDoubleLeftIcon } from "@twilio-paste/icons/esm/ChevronDoubleLeftIcon";
 import { ChevronDoubleRightIcon } from "@twilio-paste/icons/esm/ChevronDoubleRightIcon";
 import { MoreIcon } from "@twilio-paste/icons/esm/MoreIcon";
-import { SidebarPushContentWrapper } from "@twilio-paste/sidebar";
-import { useTheme } from "@twilio-paste/theme";
-import { Topbar } from "@twilio-paste/topbar";
-import { useUID } from "@twilio-paste/uid-library";
-import * as React from "react";
-
+import { Separator } from "@twilio-paste/separator";
 import {
   SidePanel,
   SidePanelBody,
   SidePanelButton,
   SidePanelContainer,
+  SidePanelContext,
   SidePanelHeader,
   SidePanelHeaderActions,
   SidePanelPushContentWrapper,
-} from "../src";
+} from "@twilio-paste/side-panel";
```

Comment:
question: why this change?

Reply:
I reviewed the other stories and they all used the same import format. I only now realize that is for all other components except the package it is changing. I will revert this